index.html

```diff
@@ -432,6 +433,7 @@ <h2 class="contact-title">Contact me</h2>
         </form>
       </footer>
     </main>
```

Comment:
https://github.com/NatiG25/Portifolio/blob/a9790ad1785a44b254e97410eef1457263da55cc/index.html#L126-L314


- [ ] Great job creating all the content of popup modal dynamically with the information from the projects array :+1:  Based on requirements (all the HTML of the section is created when the page loads). The JavaScript code should have an array of objects to store JavaScript objects containing the information for each project and his popup. Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie: eg:
````javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
````

![image](https://user-images.githubusercontent.com/57421823/151015393-2368bdbc-0f28-46f4-9357-b17c3811e52d.png)

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty

Reply:
I'm sorry I don't quite understand your suggestion. I already did that in the folder script/ popup.js.

Reply:
const projects = [
  {
    title: 'Tonic',
    src: './image/Snapshoot-Portfolio.png',
    description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea"],
    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
    seeLivelink: '#',
    seeSourceLink: '#',
  },

  {
    title: 'Multi-Post Stories',
    src: './image/Snapshoot-Portfolio.svg',
    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
    seeLivelink: '#',
    seeSourceLink: '#',
  },
  {
    title: 'Tonic',
    src: './image/Snapshoot-3Portfolio.svg',
    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
    seeLivelink: '#',
    seeSourceLink: '#',
  },
  {
    title: 'Multi-Post Stories',
    src: './image/Snapshoot-4Portfolio.svg',
    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
    seeLivelink: '#',
    seeSourceLink: '#',
  },
];

Reply:
This is already on my pop.js file.